src/app/app.component.spec.ts

```diff
@@ -6,14 +6,14 @@ import {AppConfig} from './config/app.config';
 import {APP_INITIALIZER} from '@angular/core';
 import {BrowserModule} from '@angular/platform-browser';
 import {FormsModule} from '@angular/forms';
-import {HttpModule} from '@angular/http';
+import {HttpClientModule} from '@angular/common/http';
 import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
 import {ResourceService} from './services/resource.service';
-import {EndpointService} from './services/endpoint.service';
+import {AuthenticationService} from './services/authentication.service';
```

Comment:
just thought about this: do we have an idea how to deal with authorization in future? create another service called authorization.service? or manage both authentication and authorization in the same service?

Reply:
I am not sure about that. 
The thing is that while it is relatively easy to have common authentication across back end systems, authorisations are more system-specific.
Which means that within Glowing Bear handling authentication is fine, but authorisations are more tricky.
Because of that, in the future there will be a choice that will need to be made about authorisations:
- find a common ground across back end systems and enforce them in GB
- let the back end systems deal with the authorisation

IMO this will require more thinking.